lib/howl/io/process.moon

```diff
@@ -147,13 +175,15 @@ class Process
       stderr = {}
       on_stderr = (err) -> stderr[#stderr + 1] = err
 
-    stdout_done = on_stdout and dispatch.park "process-wait-stdout-#{@pid}"
-    stderr_done = on_stderr and dispatch.park "process-wait-stderr-#{@pid}"
-    pump_stream(@stderr, on_stderr, stderr_done, true) if on_stderr
-    pump_stream(@stdout, on_stdout, stdout_done) if on_stdout
+    @stdout_done = on_stdout and dispatch.park "process-wait-stdout-#{@pid}"
```

Comment:
Oddly enough, the code seems to work now without my changes. I don't actually know why exactly it didn't work before, but I'm guessing something else I did fixed this...